### DISCUSSION TITLE
Using TikZ in Jekyll posts

### DISCUSSION THREAD

**Discussion by cortix** [mmistakes/minimal-mistakes#3937] [Ideas]
I've been looking for ways to use TikZ on Jekyll for a while. Finally, I came across an article written on this topic. But I couldn't quite understand how to use it in the minimal mistakes template.

This is the post I mentioned.
https://polbarrachina.com/2022/05/23/latex-and-tikz-in-jekyll-posts.html

I am able to use Latex but not able to use TikZ. I think this guy found a solution. It would be really nice if we could use this feature in the minimal mistakes template.

**Comment by iBug** [comment]
Nice find, but this is completely out of the scope of a theme.
